ftw/keywordwidget/vocabularies.py

```diff
@@ -91,6 +91,9 @@ def __getattr__(self, attr):
             return getattr(self, attr)
         return getattr(self._source, attr)
 
+    def __iter__(self):
+        return iter(self._source)
```

Comment:
Shouldn't this include terms from `self.instance_vocabulary`?

Reply:
I also asked myself that question. But for now just all magic methods are delegated to the source, so we'd have the same question with `__contains__` for example. The truth is that it would be a bit of a pain to implement an iterator that goes over both the source and the instance vocabulary and that we do not need it for now. We do not iterate over the vocabulary in the same request where we add new terms...

Reply:
I'll fix the `__iter__` and implement the `__contains__`